math-scala/src/main/scala/org/apache/mahout/math/algorithms/regression/LinearRegressorModel.scala

```diff
@@ -70,17 +76,43 @@ trait LinearRegressorFitter[K] extends RegressorFitter[K] {
     val varCovarMatrix = invDegFreedomKindOf * ete(0,0) * drmXtXinv
     val se = varCovarMatrix.viewDiagonal.assign(SQRT)
     val tScore = model.beta / se
-    val tDist = new org.apache.commons.math3.distribution.TDistribution(n-k)
+    val tDist = new TDistribution(n-k)
+
+    // This is the residual sum of squares for just the intercept
+    //println(" drmTarget.ncol) = " +  drmTarget.ncol)
+    val interceptCol = model.beta.length - 1
+    val targetMean: Double = drmTarget.colMeans().get(0)
+    val rssint: Double = ((drmTarget - targetMean  ).t %*% (drmTarget - targetMean)).zSum()
+    val rssmod: Double = ete.zSum()
+    // ete above is the RSS for the calculated model
+
+    //println(" model.beta(0) = " +  model.beta(0))
+    //println(" model.beta(interceptCol) = " +  model.beta(interceptCol))
+    //println("rssint = " + rssint)
+    //println("rssmod = " + rssmod)
+
+    val groupDof = X.ncol-1
+    val fScore = ((rssint - rssmod) / groupDof) / ( rssmod / (X.nrow - groupDof- 1 ))
```

Comment:
this needs to be refactored as a fitness test- the rationale is unlike R for larger datasets the user will not invoke calcStandardStatistics, and instead will execute the tests she needs a la cart.